thirdeye/thirdeye-frontend/app/pods/services/api/anomalies/service.js

```diff
@@ -0,0 +1,81 @@
+import Service from '@ember/service';
+import { inject as service } from '@ember/service';
+import EmberObject from '@ember/object';
+import { assert } from '@ember/debug';
+/**
+ * @type {Ember.Service}
+ * @summary This service provides all the api calls for anomalies related data. An `Ember.Service`
+   is a long-lived Ember object that can be made available in different parts of your application.
+ * @example anomaliesApiService: service('services/api/anomalies');
+ */
+export default Service.extend({
+  store: service('store'),
+  queryCache: service('services/query-cache'),
+  init() {
```

Comment:
Do we need this `init()` if we're only calling `this_super()`?

Reply:
Nope. You're right. I had something there before but removed it. Let me remove this init now. Thanks!